README.md

```diff
@@ -1852,7 +1852,43 @@ export default class ReactForm extends React.Component{
 
 ### Vue.js
 
-> TODO
+You can use the `v-model` directive to create **two-way data bindings** on form `input` and `textarea` elements. It automatically picks the correct way to update the element based on the input type. Although a bit magical, `v-model` is essentially syntax sugar for updating data on user input events, plus special care for some edge cases.
```

Comment:
Could you add this paragraph to the `Data binding` section?

Reply:
I Remove it from here and I add it to the `Data binding` section?

Reply:
Just the text about `v-modal` should be moved to `Data binding` -> `Vue.js`. The rest stays here.